silkrpc/json/stream.cpp

```diff
@@ -16,12 +16,94 @@
 
 #include "stream.hpp"
 
+#include <algorithm>
+#include <string>
+#include <vector>
+
+#include <boost/asio/write.hpp>
+
 namespace json {
-boost::asio::awaitable<void> Stream::flush() {
-    co_return;
+
+static std::uint8_t OBJECT_OPEN = 1;
```

Comment:
Rename `OBJECT_OPEN` as `kObjectOpen` to follow naming conventions

Reply:
OK, done